src/main/resources/api/api.yaml

```diff
@@ -150,6 +150,167 @@ paths:
   #
   # Questions
   #
+  /v1/Questions:
+    parameters:
```

Comment:
As I could see that parameter applies only to get request, in post we will pass topic in body, so we do not need that required parameter. 
So, moving it into get solves problem. Am I right?